oauthenticator/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .oauth2 import *
 from .github import *
 from .bitbucket import *
+from .mediawiki import *
```

Comment:
Alright! In that case, I wonder if I should call the class mediawiki.OAuthenticator, since mediawiki.MWOAuthenticator seems redundant.